lib/expressir/liquid/repository_drop.rb

```diff
@@ -5,16 +5,22 @@
 module Expressir
   module Liquid
     class RepositoryDrop < ::Expressir::Liquid::ModelElementDrop
-      def initialize(model)
+      def initialize(model, selected_schemas: nil, options: {}) # rubocop:disable Lint/MissingSuper
         @model = model
+        @selected_schemas = selected_schemas
+        @options = options
         super
       end
 
       def schemas
         return [] unless @model.schemas
 
         @model.schemas.map do |item|
-          ::Expressir::Liquid::Declarations::SchemaDrop.new(item)
+          ::Expressir::Liquid::Declarations::SchemaDrop.new(
```

Comment:
You could make a new method RepositoryDrop.filter that returns the selected SchemaDrop objects.